crypto/keys/keys_test.go

```diff
@@ -324,3 +324,25 @@ func TestGenerateKey(t *testing.T) {
 		}
 	}
 }
+
+func TestMarshalPrivateKey(t *testing.T) {
+	for _, tc := range []struct {
+		pem     string
+		typeURL string
+	}{
+		{ecdsaPrivateKey, "type.googleapis.com/keyspb.PrivateKey"},
+		{rsaPrivateKey, "type.googleapis.com/keyspb.PrivateKey"},
+	} {
+		k, err := NewFromPrivatePEM(tc.pem, "")
+		if err != nil {
+			t.Errorf("NewFromPrivatePEM(%v): %v", tc.pem, err)
```

Comment:
Add `continue` after t.Errorf, otherwise it'll probably panic on failed tests. Ditto for others.